src/_posts/2014-02-19-appveyor-20-dedicated-build-vms-parallel-testing-nuget-deployment.md

```diff
@@ -23,7 +23,7 @@ deployment.
 * [NuGet hosting](#nuget)
 * [Build matrix](#build-matrix)
 * [Parallel testing](#parallel-testing)
-* [Deployment to multiple environments](#deployment)
+* [Deployment to multiple environments](#deploying)
```

Comment:
There are two posts with a "Deployment" header in, that are now on the same page (page 6). Presumably they were in different pages before.

Jekyll uses the header text in html name attributes, so they must be unique on the page.

The html linter complained about this so I changed one heading from "Deployment" to "Deploying".

Then the link checker complained so I changed the line above to fix it.

Cheers

Cedd